packages/core/src/rules/common/spec.ts

```diff
@@ -110,6 +110,15 @@ export const OasSpec: Oas3Rule | Oas2Rule = () => {
             }
           }
         }
+
+        if (propName === 'example' && node.type === 'array') {
```

Comment:
Why do we have it only for `array` type? Doesn't it make sense to validate all types?

Reply:
Good question -- we should validate all example types.

Reply:
will do for all types in one rule